src/loaders/index.js

```diff
@@ -26,19 +26,19 @@ const initLoaders = async (app) => {
   });
 
   app.use((err, req, res, next) => {
-    const error =
-      process.env.NODE_ENV === "development" || err.status
-        ? err
-        : new Error("Internal server error");
+    logger.error(err.toString());
```

Comment:
여기서 한 번만 logger를 남기는 방식으로 수정해주셨군요! 보기에 훨씬 깔끔한 것 같습니다!

근데, 예전에 제가 여기에 남겼을 땐, 무슨 에러가 발생했는지는 알 수 있지만, 어느 파일의 어느 로직에서 에러가 났는지 알 수 없었는데,
그 문제를 해결할 수 있는 방법이 있을까요?

Reply:
```suggestion
    logger.error(err);
```
에러객체를 활용하면 될듯하네요
err.stack도 로그에 남긴다던지...

Reply:
네넵!